text/0032-access_command.md

```diff
@@ -0,0 +1,215 @@
+* move `publish` and `unpublish` into access, changing both commands to edit _only_ `list` visibility (not push data)
+* add an `info` command under access to show access info for a dataset
+* pushing to the registry automatically runs "publish". dropping from the registry automatically unpublishes
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+
+### The Access Subcommand:
+Here's the proposed access subcommand `help`:
+
+```
+$ qri access --help
+Access controls visibility and permissions for datasets.
+
+Usage:
+  qri access [command]
+
+Available Commands:
+  info           show access control details for a dataset
+  publish        list dataset for others to see and pull
+  unpublish      unlist dataset, removing it from public visibility
+```
+
+Publish used to have a note talking about how it pushes data. This RFC assumes that work is now handled by `qri push`, so now publish only controls `list` visibility:
+
+```
+$ qri access publish --help
+Publish makes your dataset publically visible to others. While online, peers 
+that connect to you can only see datasets and versions that you've published. 
+Publishing a dataset always makes all previous history entries available, and 
+any updates to a published dataset will be immediately visible. Published 
+datasets will show up in feeds.
+
+New datasets are *not* published by default, remaining unlisted until published.
+
+Pushing to the registry requires a dataset be either encrypted or published.
+By default pushing a dataset to the registry also publishes.
+
+Usage:
+  qri access publish [DATASET] [DATASET...] [flags]
+
+Examples:
+  # Publish a dataset so others can see it with "qri list":
+  $ qri access publish me/dataset
+
+  # Publish a few datasets:
+  $ qri access publish me/dataset me/other_dataset
+```
+
+Unbpublish used to be a flag. In this RFC it's bumped up to a subcommand of it's own, to match other reciprocal commands like `push/pull`. 
+
+```
+$ qri access unpublish --help
+Unpublish "unlists" a dataset. An unlisted dataset will not show up when
+browsing dataset lists or feeds. Others can still access an unpublished dataset
+if they know it's name.
+
+Unpublished datasets cannot be pushed to the registry, but can be pushed to
+peers that accept unpublished datasets from you.
+
+Usage:
+  qri access unpublish [DATASET] [DATASET...] [flags]
+
+Examples:
+  # unpublish a dataset, removing it from any lists or feeds
+  $ qri access unpublish me/dataset
+
+  # Publish a few datasets:
+  $ qri access unpublish me/dataset me/other_dataset
+ 
+Flags:
+  -r, --remove     issue "remove all" commands to any published remotes
+```
+
+
+Publish & Unpublish on their own may not seem like they warrant being moved into a subcommand. To understand the justification for the `access`, this is how `access` _might_ look one day, after we've landed encryption and role-based access control (RBAC) for datasets:
```

Comment:
I did not like the name _access_ until I got to here.

Added comment at the top to clarify in summary.